airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2871,6 +2917,33 @@ components:
       properties:
         consentUrl:
           type: string
+    OAuthSourceRequestBody:
```

Comment:
we should also have additionalProperties=true because there may be additional properties depending on the API